src/script/common/c_content.cpp

```diff
@@ -986,6 +1009,7 @@ void read_content_features(lua_State *L, ContentFeatures &f, int index)
 	} else if(lua_isnil(L, -1)) {
 		f.liquid_move_physics = f.liquid_type != LIQUID_NONE;
 	} else {
+		// TODO: should be an error
 		errorstream << "Field \"liquid_move_physics\": Invalid type!" << std::endl;
```

Comment:
Idea: also provide a node name here such that modders can find the culprit more easily.

Reply:
The idea is that a future PR (soon) would directly turn this into an error so not worth editing it now.